packages/sycamore-macro/src/view/codegen.rs

```diff
@@ -433,16 +433,20 @@ impl Codegen {
                     #[cfg(target_arch = "wasm32")]
                     ::sycamore::reactive::create_effect(#cx, {
                         let __el = ::std::clone::Clone::clone(&__el);
+                        let #expr = ::std::clone::Clone::clone(&#expr);
                         move ||::sycamore::generic_node::GenericNode::set_property(
                             &__el,
                             #prop,
                             &#convert_into_jsvalue_fn,
                         )
                     });
                     ::sycamore::generic_node::GenericNode::event(&__el, #cx, #event_name,
-                        ::std::boxed::Box::new(|event: ::sycamore::rt::Event| {
+                    {
+                        let #expr = ::std::clone::Clone::clone(&#expr);
+                        ::std::boxed::Box::new(move |event: ::sycamore::rt::Event| {
                             #expr.set(#convert_from_jsvalue_fn);
-                        }),
+                        })
+                    },
```

Comment:
Introduces implicit cloning of signals in the same way as for the builder API.

This solves issues like this:
```rust
let signal = create_rc_signal(String::new());
view! { cx,
    input(bind:value=signal)
    (signal.get())
}
```

Note that order will matter. Binding a signal after reading it will still give a "value borrowed after move" error.